contrib/terraform/openstack/group_vars/gfs-cluster.yml

```diff
@@ -0,0 +1 @@
+ansible_ssh_common_args: '-o ProxyCommand="ssh -o StrictHostKeyChecking=no -W %h:%p -q ubuntu@193.62.54.58"' 
```

Comment:
Apologies.. I git added that file by mistake, it is dynamically generated by terraform based on a floating ip used when provisioning the k8s nodes, on each terraform apply execution. Will remove the file from the commit. Thanks for spotting it!